kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternFieldsController.java

```diff
@@ -84,8 +111,18 @@ public class PatternFieldsController {
     @FXML
     private StackPane selectedMeaningStackPane;  // StackPane to hold the dropped/selected meaning item
 
+    @FXML
+    private ComboBox<Integer> fieldOrderComboBox = new ComboBox<>();
+
+    private ObservableList<Integer> fieldOrderValues;
+
     @FXML
     private void initialize() {
+        // initially set the combo box to 1
+        setUpObservableFieldOrder();
```

Comment:
Can you add this to the view model? 
Something like the following:
```java
patternFieldsViewModel.setPropertyValues("fieldOrder", List.of(1));
```

Reply:
removed for now... but will add this later, thanks